src/java.base/share/classes/java/net/NetMulticastSocket.java

```diff
@@ -652,6 +652,7 @@ public Set<SocketOption<?>> supportedOptions() {
 
     @Deprecated
     @Override
+    @SuppressWarnings("removal")
```

Comment:
Instead of adding `@SuppressWarning("removal")` here (and at other places below) - would it make sense to update the `@Deprecated` annotation and add `forRemoval = true`?

Reply:
We will still have to add the `@SuppressWarning("removal")` to these methods otherwise the compilation generates a warning which fails the build:

```
java.base/share/classes/java/net/NetMulticastSocket.java:655: warning: [removal] setTTL(byte) in MulticastSocket has been deprecated and marked for removal
    public void setTTL(byte ttl) throws IOException {
                ^
java.base/share/classes/java/net/NetMulticastSocket.java:673: warning: [removal] getTTL() in MulticastSocket has been deprecated and marked for removal
    public byte getTTL() throws IOException {
                ^
error: warnings found and -Werror specified
1 error
2 warnings

```
I went ahead and updated these internal `NetMulticastSocket` and `DatagramSocketAdaptor` classes to add the `forRemoval` attribute to the `@Deprecated` annotation to keep it consistent.

Reply:
Ah - thank you. I had hoped that adding forRemoval = true to the annotation would allow us to get rid of the SuppressWarning.